lib/smart_proxy_container_gateway.rb

```diff
@@ -1,2 +1,4 @@
 require 'smart_proxy_container_gateway/version'
+require 'smart_proxy_container_gateway/database'
 require 'smart_proxy_container_gateway/container_gateway'
+require 'smart_proxy_container_gateway/container_gateway_main'
```

Comment:
Technically you could implement `load_classes` in `Proxy::ContainerGateway::Plugin`, but I think this is fine as well. It makes testing easier. The only downside is that it's slightly slower when you have the plugin installed but not enabled, but that's really small and only a tiny number of installations.